Sources/PackageLoading/MinimumDeploymentTarget.swift

```diff
@@ -30,6 +30,12 @@ public struct MinimumDeploymentTarget {
         return PlatformVersion(versionString)
     }
 
+    static func computeXCTestMinimumDeploymentTarget(with runResult: ProcessResult) throws -> PlatformVersion? {
+        let sdkPath = try AbsolutePath(validating: try runResult.utf8Output().spm_chuzzle() ?? "/")
```

Comment:
Isn't this always going to result in a path that doesn't exist?  Or is that inevitable if we can't determine the path using xcrun?

Reply:
Yah, the idea is that we are taking the fallback version if `xcrun` fails. Do you think we should make that more explicit?

Reply:
Maybe just a comment that it is known that `/Developer/Library/Frameworks/XCTest.framework/XCTest` will never exist, but that that involves the fallback path.  Perhaps it's just a nitpick, but when I first read it, it seemed slightly odd that the two lines here would always result in a path that is expected to not exist.

Reply:
Actually, since this is now its own method, it might be simpler to just convert this to a `guard` statement. I didn't do this previously because I wanted to have a single exit point for the fallback in case that ever got more complex to decide, but that consideration doesn't exist here anymore.

Reply:
Updated this to a `guard` statement now.